src/types.ts

```diff
@@ -444,3 +447,11 @@ export enum CheckListViewType {
   Card,
   List,
 }
+
+export enum HTTPCompressionAlgo {
+  None = '',
```

Comment:
nit: None is capitalized but rest are not. 
```suggestion
  none = '',
```

ps: nits are trivial comments, feel free to ignore them.